app/controllers/api/v2/green_lanes/goods_nomenclatures_controller.rb

```diff
@@ -15,6 +15,7 @@ def show
               applicable_measures
               applicable_category_assessments
               applicable_category_assessments.exemptions
+              applicable_category_assessments.geographical_area
```

Comment:
can this also include excluded areas thanks

Reply:
done